discuss/customizations/mail_activity_history.rst

```diff
@@ -42,4 +42,16 @@ analyzed using custom filters in list view.
 .. image:: media/activity_history_menu.png
     :align: center
 
+Analyze current Activities
```

Comment:
This part is unnecessary, because it is already incuded here https://doc.focusate.eu/discuss/customizations/mail_activity_extended.html